scripts/pre-release.js

```diff
@@ -6,15 +6,12 @@ const path = require('path');
 
 const trackingConfigFilePath = path.join(process.cwd(), getTrackingConfigFileName());
 
-// don't release without tracking keys!
-if (!process.env.SEGMENT_WRITE_KEY) {
-  throw new Error('SEGMENT_WRITE_KEY env var not set');
-} else if (!process.env.SENTRY_DSN) {
+// don't release without Sentry key!
+if (!process.env.SENTRY_DSN) {
   throw new Error('SENTRY_DSN env var not set');
 }
 
 const trackingConfig = {
-  segmentWriteKey: process.env.SEGMENT_WRITE_KEY,
```

Comment:
We might want to keep this to overwrite the behavior of sending everything to our prod setup by default (see other comment in this PR review).